radio/src/keys.h

```diff
@@ -148,14 +148,14 @@ inline bool IS_KEY_EVT(event_t evt, uint8_t key)
 
 inline bool IS_NEXT_EVENT(event_t evt)
 {
-  return evt == EVT_KEY_FIRST(KEY_DOWN) || evt == EVT_KEY_REPT(KEY_DOWN) ||
+  return evt == EVT_KEY_FIRST(KEY_RIGHT) || evt == EVT_KEY_REPT(KEY_RIGHT) ||
          evt == EVT_KEY_FIRST(KEY_MINUS) || evt == EVT_KEY_REPT(KEY_MINUS) ||
          evt == EVT_ROTARY_RIGHT;
 }
 
 inline bool IS_PREVIOUS_EVENT(event_t evt)
 {
-  return evt == EVT_KEY_FIRST(KEY_UP) || evt == EVT_KEY_REPT(KEY_UP) ||
+  return evt == EVT_KEY_FIRST(KEY_LEFT) || evt == EVT_KEY_REPT(KEY_LEFT) ||
```

Comment:
Actually, why did you change this? As up/down was correct for menu navigation on the t-lite (and also the LR3pro I think). Hence #4917

Reply:
To fix the left right scrolling on the SF/GF page for 9x nav - I fixed this first and meant to come back and check that it did not affect other things (which it clearly did). Thanks for picking it up - the fix in 4917 looks better.

Reply:
Yeah, I thought that might be the case when digging a bit deeper...  I suspect it is has been broken somewhere else, but that should hopefully do for now to make most all the use cases work. :crossed_fingers: